core-web/libs/edit-content/src/lib/components/dot-edit-content-sidebar/components/dot-edit-content-sidebar-workflow/dot-edit-content-sidebar-workflow.component.ts

```diff
@@ -7,9 +7,21 @@ import { DialogModule } from 'primeng/dialog';
 import { DropdownModule } from 'primeng/dropdown';
 import { SkeletonModule } from 'primeng/skeleton';
 
-import { DotCMSWorkflowStatus } from '@dotcms/dotcms-models';
+import { DotCMSWorkflowAction } from '@dotcms/dotcms-models';
 import { DotMessagePipe } from '@dotcms/ui';
 
+import { DotWorkflowState } from '../../../../models/dot-edit-content.model';
+
+interface WorkflowSelection {
+    schemeOptions: SelectItem[];
+    noWorkflowSelectedYet: boolean;
+}
+
+const DEFAULT_WORKFLOW_SELECTION: WorkflowSelection = {
+    schemeOptions: [],
+    noWorkflowSelectedYet: false
```

Comment:
instead of use a negation variable, use something like `isWorkflowSelected` ?

Reply:
changed